.github/workflows/OS/mary.yml

```
@@ -0,0 +1,35 @@
+name: Summarize new issues
+
+on:
+  issues:
+    types: [opened]
+
+jobs:
+  OS mary:
```

actionlint (1.7.12)

[error] invalid job ID "OS mary". job ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)